js-api-doc/options.d.ts

```diff
@@ -352,10 +367,10 @@ export interface Options<sync extends 'sync' | 'async'> {
    *
    * This option has no effect if {@link sourceMap} is `false`.
```

Comment:
The docs here should now say "When" rather than "Whether". We should also indicate that passing a boolean is deprecated.